### PR TITLE
Refactor/submitclaims claims

### PR DESCRIPTION
**Pull request by ccamel**
Addresses #580.

## Summary by CodeRabbit

- **New Features**
  - Updated documentation to reflect the new `submit_claims.claims` parameter for enhanced data integrity.

- **Refactor**
  - Renamed the `metadata` field to `claims` in the `SubmitClaims` message structure for clarity and consistency.

### CODE REVIEW

File: contracts/axone-dataverse/src/contract.rs [hunk 1]
```diff
@@ -75,7 +75,7 @@ pub fn execute(
     nonpayable(&info)?;
     match msg {
         ExecuteMsg::SubmitClaims {
-            metadata,
+            claims: metadata,
```
[the remark this answers is not in the document]
Comment (ccamel) [reply contracts/axone-dataverse/src/contract.rs]: Sure. I'm going to align the terminology throughout the code.

**Review by bdeneux — APPROVED** [review]
Looks good 👍

**Review by amimart — APPROVED** [review]
I propose to finish all the security audit related fixed before merging this one, so we can make a proper release of the contracts with only security fixed before breaking changes the APIs, if that's ok :)

**Comment by ccamel** [comment]
Alright. Now that version [6.0.0](https://github.com/axone-protocol/contracts/releases/tag/v6.0.0) has been released, including all the security fixes from the audit, we can proceed with merging this PR.

**Comment by amimart** [comment]
@ccamel I think we can resurrect this one :)

**Pull request merged**